lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -239,9 +239,9 @@ def infer(
                     )[0]
                 )
             for result in results:
-                result["scores"] = result["scores"].cpu().detach().numpy()
-                result["labels"] = result["labels"].cpu().detach().numpy()
-                result["boxes"] = result["boxes"].cpu().detach().numpy()
+                result["scores"] = result["scores"].tolist()
+                result["labels"] = result["labels"].tolist()
+                result["boxes"] = result["boxes"].tolist()
```

Comment:
Is this part of the test?  Is this a set of fixes to make the test pass?  If so, do we still need #418?